src/main/drivers/accgyro_mpu.c

```diff
@@ -211,6 +211,10 @@ static void mpu6050FindRevision(void)
 
 extiCallbackRec_t mpuIntCallbackRec;
 
+#define DEBUG_MPU_DATA_READY_INTERRUPT
```

Comment:
Oops. Apologies. I was using accgyro_mpu.c as a debugging aid. Never meant to be a part of the PR.